cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -72,16 +72,16 @@ void MapBuilderBridge::SerializeState(const std::string& stem) {
 void MapBuilderBridge::WriteAssets(const string& stem) {
   const auto all_trajectory_nodes =
       map_builder_.sparse_pose_graph()->GetTrajectoryNodes();
-  // TODO(yutakaoka): Add multi-trajectory support.
   CHECK_EQ(trajectory_options_.count(0), 1);
```

Comment:
Add a comment that this only checks that trajectory 0 options exist which we will use here? Seems easy to misunderstand.